README.md

```diff
@@ -51,7 +51,7 @@ On successful execution of `make build`, the output binary `csi-proxy.exe` will
 csi-proxy.exe can be installed and run as binary or run as a Windows service on each Windows node. See the following as an example to run CSI Proxy as a web service.
 ```
     $flags = "-windows-service -log_file=C:\etc\kubernetes\logs\csi-proxy.log -logtostderr=false"
```

Comment:
If the node is created with kube-up.sh from the kubernetes/kubernetes repo I believe that https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/k8s-node-setup.psm1#L449-L459 will redo the setup and start CSI Proxy again if the node is restarted

After this change we'd need to do the setup only once and then on reboot it'd be started automatically, is my understanding of the setup being done only once correct?

Reply:
Looks like https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/k8s-node-setup.psm1#L449-L459 is independent of the ps1 script but it can certainly use the same strategy.

Reply:
Should a new PR be sent for this repo similar to this change?

Reply:
not really, it was more of a general question about in which scenarios we would need to put this flag in a prod environment, the kube-up.sh script are only used to create dev clusters and e2e clusters